static/assets/js/entity-reference/app.js

```diff
@@ -34,6 +42,10 @@
             var whiteList = this.whiteList ? this.whiteList.split(',') : [];
             var blackList = this.blackList ? this.blackList.split(',') : [];
             this.entities = getEntitiesFromSwagger(swagger, whiteList, blackList);
+
+            $timeout(function() {
+                $anchorScroll();
+            });
```

Comment:
What does this do? Probably worth leaving a comment or give the function a description name.